menu.js

```diff
@@ -1,5 +1,48 @@
 const menu = {
-  // TODO
+  _courses: {
+      mains: [],
+      desserts: [],
+      appetizers: []
+  },
+  get courses() {
+    return this._courses;
+  },
+  addDishToCourse(courseName, dishName, dishPrice) {
+     let dishGroup = {
+       name: dishName,
+       price: dishPrice
+     };
+     switch(courseName) {
+       case 'mains':
+            menu._courses.mains.push(dishGroup);
+            break;
+       case 'desserts':
+            menu._courses.desserts.push(dishGroup);
+            break;
+       case 'appetizers':
+            menu._courses.appetizers.push(dishGroup);
+            break;
+        default:
+            console.log('Error courses');
+     }
+  },
+  getRandomDishFromCourse(courseName) {
+      const randomIndex = Math.floor(Math.random() * this._courses[courseName].length);
+      return this._courses[courseName][randomIndex];
```

Comment:
You are calling `this._courses[courseName]` 2 times. I think making extra variable for its value is a good idea. Something like `courseDishes` should be fine.